docs/get-started.rst

```diff
@@ -182,6 +182,11 @@ Make sure you have the system requirements listed in the
     cd kinto/
     make serve
 
+At the end of the installation, you will be asked which backend you would like to use:
+
+    $ Select the backend you would like to use: (1 - postgresql, 2 - redis, default - memory)
+
+If you don't know, just push "enter" to choose the default Memory backend, when you will need another one you will know it :)
```

Comment:
I guess we need to help the user to "change it later".
I suppose we can change the backends directly in `config/kinto.ini` (with [the help of the doc](https://kinto.readthedocs.org/en/latest/configuration/settings.html#backends)).

But is there a CLI to to get this question back? `Select the backend you would like to use: (1 - postgresql, 2 - redis, default - memory)` And so have a wizard to setup postgres or redit.